contentcuration/contentcuration/frontend/accounts/pages/Create.vue

```diff
@@ -241,6 +242,9 @@
       passwordConfirmRules() {
         return [value => (this.form.password1 === value ? true : this.$tr('passwordMatchMessage'))];
       },
+      passwordValidationRules() {
+        return [value =>(this.form.password1.length >= 8 ? true : this.$tr('passwordValidationMessage'))];
```

Comment:
I think simply using `value.length >= 8` should suffice as the context of the password field is that of the form element `password1`.